Update seminars (#1)
DragonflyStats/Projects: WhyR/About/proposal/requirements.Rmd

# Requirements
<!-- 
An idea of what is required to make the project actually happen
-->

## People

> This work will be completed by gwynn, Kevin, Ritwik, Lorna, and others who volunteer.

Reviewer: I would maybe add their actual GitHub '@'s to this.

<!--
Who needs to be involved, what's the proposed structure, what will it take to get their involvement?
-->

## Processes

> We will follow the RConsortium COC and update it for online meetings or zoom events. We anticipate having meetings every 2 months about progress and quarterly ISC meetings.

Reviewer: ```suggestion
We will follow the RConsortium Code of Conduct (CoC) and update it for online meetings or Zoom events. We anticipate having meetings every 2 months about progress and quarterly ISC meetings.
```

<!-- 
What processes need to be put in place e.g. codes of conduct, regular ISC meetings, handover to the community at large?
-->

## Tools & Tech & Development

> To complete this research we will need access to books about R and S. The books Statistical models in S (1992), Mixed-Effects Models in S and S-PLUS (2009), Regression Modeling Strategies (2013), An R and S-Plus Companion to Applied Regression (2002), and Modern Applied Statistics with S (2002) are all candidates and there may be others.

Reviewer: ```suggestion
To complete this research we will need access books about both R and S. The possible initial list of books would include the following:  
* Statistical models in S (1992)
* Mixed-Effects Models in S and S-PLUS (2009)
* Regression Modeling Strategies (2013)
* An R and S-Plus Companion to Applied Regression (2002)
* Modern Applied Statistics with S (2002)
```

<!--
What is going to be needed to deliver this project? 

Will cloud computing be used - if yes are there are necessary components that will be deciding factors between providers?

Are there tools or tech that don't exist that will be produced to facilitate the project?
-->

## Funding

> We anticipate the costs of this project will be in building a website and acquiring materials.

Reviewer: ```suggestion
We anticipate the main costs of this project will be in building a website and acquiring research materials.
```
